litespi/modules/modules.py

```diff
@@ -3,3 +3,56 @@
 from litespi.ids import SpiNorFlashManufacturerIDs
 
 # Define non-generated SPI NOR chips here
+
+# MX25L6406E Datasheet:
+# https://www.macronix.com/Lists/Datasheet/Attachments/7370/MX25L6406E,%203V,%2064Mb,%20v1.9.pdf
+class MX25L6406E(SpiNorFlashModule):
+
+    manufacturer_id = SpiNorFlashManufacturerIDs.MACRONIX
+    device_id = 0x2017
+    name = "mx25l6406e"
+
+    total_size  =    8388608   # bytes
+    page_size   =        256   # bytes
+    total_pages =      32768
+
+    supported_opcodes = [
+        SpiNorFlashOpCodes.READ_1_1_1,
+        SpiNorFlashOpCodes.READ_1_1_1_FAST,
+        SpiNorFlashOpCodes.READ_1_1_2,
+        SpiNorFlashOpCodes.PP_1_1_1,
+    ]
+    dummy_bits = 8
+
+
+# W25Q512JV Datasheet:
```

Comment:
These should be in the docstrings so they appear in the generated documentation output.